### ISSUE TITLE
1 sentence will be splitted into 3 sentences for training data?

### ISSUE THREAD

**Issue by YuHengKit** [sysulic/TAS-BERT#2]
Hi there, I read your paper and I have some questions:
1. Is every sentence will be split into 3 sentences before fed into BERT as figure shown in paper?
![image](https://user-images.githubusercontent.com/42687320/94160695-78f4c980-feb7-11ea-8757-f1303a4b0e1d.png)

2. Can you explain a little bit about the output for aspect and sentiment with CRF/Softmax decoder in simple word? How to differentiate the output for aspect and for sentiment? 

Thanks for your effort and hope to hear from you soon.
